docs/UserGuide.md

```diff
@@ -41,146 +37,167 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
-### 2.1. Viewing help : `help`
+### Viewing help : `help`
 
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### Adding a task: `addTask`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the task manager<br>
+Format: `addTask TASKNAME d/DEADLINE DATE [t/DEADLINE TIME] [p/PRIORITY] [#TAG]...`
 
-> Persons can have any number of tags (including 0)
+> Tasks can have any number of tags (including 0)
+> Tasks can only have one priority
+> Tasks can only have one deadline time
+> Tasks must have a deadline date
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `addTask completeUserGuide d/28-Feb-2017 t/2400 p/high #Important`
 
-### 2.3. Listing all persons : `list`
+### Adding a floating task: `addFTask`
+
+Adds a floating task to the task manager<br>
+Format: `addFTask FLOATINGTASKNAME [p/PRIORITY] [#TAG]...`
+
+> Floating Tasks can have any number of tags (including 0)
+> Floating Tasks can only have one priority
+> Floating Tasks cannot have any deadlines 
+
+Examples:
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+* `addFTask reachDiamondRank p/low #Overwatch`
 
-### 2.4. Editing a person : `edit`
+### Adding an event: `addEvent`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Adds an event to the task manager<br>
+Format: `addEvent EVENTNAME sd/START DATE ed/END DATE [st/START TIME] [et/END TIME] [p/PRIORITY]  [#TAG]...`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+> Events can have any number of tags (including 0)
+> Events can only have one priority
+> Events must have a start and end date
+> If no start time or end time specified, system will assume start at 0000 on start date and end at 2359 on end date
+
+Examples:
+
+* `addEvent finishCS3230Lab sd/01-Mar-17 ed/01-Mar-17 st/2000 et/2200 p/med #CS3230`
+
```

Comment:
Maybe you can consider combining the date and time into a single parameter to make it more user friendly.